src/components/about-us/Timeline.tsx

```diff
@@ -1,32 +1,65 @@
-import { FC } from "react";
-import "@/stylesheets/timeline.css";
+import React from "react";
+import Images from "../image/Image";
+import classNames from "classnames";
 
-type TimelineItem = {
+interface TimelineItemProps {
+  imageSrc: string; // Image source
   period: string;
-  description: string;
-};
+  bgColor: string; // Background color of the parent container
+  description: string; // Description text
+}
 
-type Props = {
-  items: TimelineItem[];
-};
+interface TimelineProps {
+  items: TimelineItemProps[];
+}
 
-const Timeline: FC<Props> = ({ items }) => (
-  <section className="px-4 pt-20 max-w-6xl mx-auto">
-    <h2 className="text-4xl font-semibold mb-4">Our history</h2>
-    <div className="timeline-container">
-      {items.map((item, i) => (
-        <div
-          key={`item-${i}`}
-          className={`timeline-item ${i % 2 === 0 ? "left" : "right"}`}
-        >
-          <p className="text-xl text-navy-600 font-bold mb-2 timeline-title">
-            {item.period}
-          </p>
-          <p>{item.description}</p>
-        </div>
-      ))}
-    </div>
-  </section>
-);
+const TimelineItem: React.FC<TimelineProps> = ({ items }) => {
+  return (
+    <>
```

Comment:
**issue (blocking):** The timeline doesn't display well on mobile.  Can we try to stack the time / card components?  Would it be helpful to have mobile-specific designs?